engine/verification/assigner/engine.go

```diff
@@ -239,19 +219,29 @@ func (e *Engine) processFinalizedBlock(ctx context.Context, block *flow.Block) {
 	}
 
 	e.metrics.OnAssignerProcessFinalizedBlock(block.Header.Height)
-	log.Info().
+	e.log.Info().
```

Comment:
```suggestion
	lg.Info().
```

Reply:
In support of the suggested change, we'd better correspond these counters with block id and block height.